shells/test/specs/restaurant-test.js

```diff
@@ -36,34 +36,31 @@ const searchFor = text => keys('input[search]', text);
   //   });
   // });
 
-  describe('demo ' + storage, () => {
-    it('restaurants', async function() {
-      await openNewArc(this.test.fullTitle(), undefined, storage);
+  describe('Restaurants Recipe storage=' + storage, () => {
+    it('restaurants suggestions', async function() {
+      openNewArc(this.test.fullTitle(), storage);
+      browser.keys('restaurants');
+      clickSuggestion('Find restaurants near you.');
+
       const search = `restaurants`;
       const findRestaurants = `[title^="Find restaurants"]`;
-      const restaurantItem = `#webtest-title`;
       const reservation = `[title*="ou are "]`;
       const calendarAction = `[particle-host="Calendar::action"]`;
-      await searchFor(search);
-      await click(findRestaurants);
+      
```

Comment:
I set this up with the data separate from the commands on purpose. It's not sacred or anything, but it seems arbitrary to just change it.